Updates to MetOffice weather integration documentation

MrHarcombe opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Updating documentation in line with PR #45149 - changes to MetOffice weather integration to provide forecast data for both 3-hourly and daily updates.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: [#45149](https://github.com/home-assistant/core/pull/45149)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

MrHarcombe committed:
Update to reflect new core functionality

Documentation now reflects the upcoming change (from PR #38725) that the weather entity will be providing forecast data for 3-hourly and daily updates.

frenck commented:
Closing PR as the parent PR has been closed.

MrHarcombe commented:
Updated to @hencoappel 's PR - can this be re-opened?

MrHarcombe commented:
> Updated to @hencoappel 's PR - can this be re-opened?

Thank you, sorry for the confusion!

frenck commented on source/_integrations/metoffice.markdown:
```diff
@@ -17,22 +17,22 @@ The `metoffice` weather platform uses the Met Office's [DataPoint API](https://w
-|sensor.**site name**_wind_direction|Sensor entity giving the current forecast wind direction.|No.|
-|sensor.**site name**_wind_gust|Sensor entity giving the current forecast maximum wind gust.|No.|
-|sensor.**site name**_speed|Sensor entity giving the current forecast wind speed.|Yes.|
+|weather.met_office\_**site name**_**frequency**|Weather entity with state of the current weather condition and attributes of temperature, humidity, wind speed and visibility as well as forecast of upcoming weather.|Yes.|
```
We should start back ticking/code fencing those entity names, as we do everywhere else in the documentation when referring to "code"-a-like word or files.

Furthermore, do these entities have a unique ID? If so, the entity IDs are not fixed, so it is probably saner to list a list of entities it provides in a descriptive manner, instead of listing am entity ID that is changeable.

MrHarcombe replied:
Hopefully that's more in line with what you meant?

MrHarcombe committed:
Removed clunky table of entities

Changes following review - entities now listed as descriptive English, rather than in a table.

frenck commented on source/_integrations/metoffice.markdown:
```diff
@@ -6,7 +6,7 @@ ha_category:
 ha_iot_class: Cloud Polling
 ha_codeowners:
-  - '@MrHarcombe'
+  - `@MrHarcombe`
```
```suggestion
  - "@MrHarcombe"
```

frenck commented on source/_integrations/metoffice.markdown:
```diff
@@ -17,22 +17,8 @@ The `metoffice` weather platform uses the Met Office's [DataPoint API](https://w
-|sensor.**site name**_wind_direction|Sensor entity giving the current forecast wind direction.|No.|
-|sensor.**site name**_wind_gust|Sensor entity giving the current forecast maximum wind gust.|No.|
-|sensor.**site name**_speed|Sensor entity giving the current forecast wind speed.|Yes.|
+The weather entities are named by concatenating `weather.m̀et_office_` with the site name and the frequency. The available sensor entities; "feels like" temperature, humidity, probability of precipitation, station name, temperature, uv index, visibility, visibility distance, weather, wind direction, wind gust and wind speed are named by concatenating `sensor.` with the site name, the entity and t ...
```
```suggestion
The weather entities are named by concatenating `weather.m̀et_office_` with the site name and the frequency. The available sensor entities; "feels like" temperature, humidity, probability of precipitation, station name, temperature, UV index, visibility, visibility distance, weather, wind direction, wind gust and wind speed are named by concatenating `sensor.` with the site name, the entity and the frequency. Only probability of precipitation, temperature, weather and wind speed are enabled by default.
```

frenck commented on source/_integrations/metoffice.markdown:
```diff
@@ -17,22 +17,8 @@ The `metoffice` weather platform uses the Met Office's [DataPoint API](https://w
-|sensor.**site name**_wind_direction|Sensor entity giving the current forecast wind direction.|No.|
-|sensor.**site name**_wind_gust|Sensor entity giving the current forecast maximum wind gust.|No.|
-|sensor.**site name**_speed|Sensor entity giving the current forecast wind speed.|Yes.|
+The weather entities are named by concatenating `weather.m̀et_office_` with the site name and the frequency. The available sensor entities; "feels like" temperature, humidity, probability of precipitation, station name, temperature, uv index, visibility, visibility distance, weather, wind direction, wind gust and wind speed are named by concatenating `sensor.` with the site name, the entity and t ...
```
So the fixed entity ID is still in here? Do the entities of this integration provide a unique ID?

avee87 replied:
Entities do provide unique IDs. How do you think we should change this wording?

MrHarcombe committed:
Update source/_integrations/metoffice.markdown

Misunderstanding on code ticks, my bad

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

MrHarcombe committed:
Update source/_integrations/metoffice.markdown

Capitalise 'UV'

Co-authored-by: Franck Nijhof <frenck@frenck.nl>
